tests/test_match.py

```diff
@@ -140,3 +158,26 @@ def test_padding(self):
         res = gmp_adopted.fit(G1, G2)
 
         assert 1.0 == (sum(res.perm_inds_ == np.arange(n)) / n)
+
+    def test_sim(self):
+        n = 150
```

Comment:
maybe the kinda thing that would be better in tutorial - but does vanilla gm not succeed here in this setting? j.w. cause i havent seen proof that the stuff in this pr works (but I trust you, dont need to go through that whole process). 

also if there's no tutorial for this PR,  can you at least make an issue to add one in the future?

Reply:
The best proof of effectiveness is found [here](https://github.com/asaadeldin11/nbview/blob/master/simaug_pr_poc.ipynb), where I recreate a figure that Youngser had made.

This figure is also the basis for the test. For rho = 1, vanilla gm would here, which is a good point, so I've changed it to rho = 0.9, and thresholded it such that vanilla gm likely wouldn't work (accuracy > 0.7).

I can add a tutorial showing how with the 0.9-ER graph's, GM+SS gives higher accuracy than vanilla gm?

Reply:
sure. but maybe the tutorial in a separate PR (can make an issue for this) cause i dont want to hold this PR up

Reply:
sounds good